src/components/colorPicker/ColorPickerDialog.tsx

```diff
@@ -212,14 +212,14 @@ class ColorPickerDialog extends PureComponent<Props, State> {
     const {hex} = this.getValidColorString(text);
 
     if (hex) {
-      _.invoke(this.props, 'onSubmit', hex, this.getTextColor(hex));
+      this.props.onSubmit?.(hex, this.getTextColor(hex));
       this.onDismiss();
     }
   };
 
   onDismiss = () => {
     this.resetValues();
-    _.invoke(this.props, 'onDismiss');
+    this.props.onDismiss?.({});
```

Comment:
Why are u passing here an empty object?

Reply:
The onDismiss props expect to get props
Changing it to be optional..